packages/core/src/integrations/requestdata.ts

```diff
@@ -92,7 +92,7 @@ const _requestDataIntegration = ((options: RequestDataIntegrationOptions = {}) =
 
       // In all other cases, use the request's associated transaction (if any) to overwrite the event's `transaction`
       // value with a high-quality one
-      const reqWithTransaction = req as { _sentryTransaction?: Transaction };
+      const reqWithTransaction = req as { _sentryTransaction?: Span };
```

Comment:
Nothing seems to set `_sentryTransaction` 🤔

Can we remove all of this functionality here?